location/templates/location.html

```diff
@@ -17,16 +17,8 @@
             $("#display_review").toggle();
         });
     });
-    let latitude = {
-        {
-            object.latitude
-        }
-    };
-    let longitude = {
-        {
-            object.longitude
-        }
-    };
+    let latitude = object.latitude;
+    let longitude = object.longitude;
```

Comment:
I think we should have the curly brackets around `object.latitude` and `object.longitude` to access the template variables because currently it does not seem to render the map properly. But I like the idea behind reformatting the code.

Reply:
hmm, unless i'm mistaken I don't think the format `let var = { value }` is valid in JS. `{}` denotes an object which I think has to have key:value pairs. If it's not displaying anymore tho maybe I misread the documentation.

Reply:
oh, turns out you were right! I misinterpreted it as incorrectly instantiating a JS object, but it is actually supposed to be a django object reference, fixed!